### PR TITLE
Iss5

### PR DESCRIPTION
**Pull request by LPMendoza**
# Description
In this pull request the next activities were made:
* Active admin setup wirth:
-Users CRUD
-Institutes CRUD
-Places CRUD
-Sections CRUD
* Sending of user key by email. 
  
 # Testing
You have to follow the next steps:
1.- Donwload the project.
2.- Put your own database keys.
3.- Run: 
```console
rake db:seeds
```
4.- Run the server and enter to localhost:3000/admin
5.- Add a new user with a real email.
  
  ## Screenshots 
image
image

### CODE REVIEW

File: app/controllers/users/confirmations_controller.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class Users::ConfirmationsController < Devise::ConfirmationsController
+  # GET /resource/confirmation/new
```
Comment (MiguelOcegueraM) [inline app/controllers/users/confirmations_controller.rb:4]: Same  here if we don't use the line delete it don't comment

**Review by MiguelOcegueraM — APPROVED** [review]
Done, good job bros

File: app/admin/institutes.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
+ActiveAdmin.register Institute do
```
Comment (OchoaHub) [inline app/admin/institutes.rb:1]: Nice gem, activeadmin helps a lot!

**Review by OchoaHub — APPROVED** [review]
I think everything's fine. Clean code, good choice of gems, I like your project. 10/10

**Review by jnegrete15 — APPROVED** [review]

**Pull request merged**